tests/import_test.go

```diff
@@ -960,7 +961,36 @@ var _ = Describe("[rfe_id:1115][crit:high][vendor:cnv-qe@redhat.com][level:compo
 			Expect(err).NotTo(HaveOccurred())
 			restarts := dv.Status.RestartCount
 			return restarts
-		}, timeout, pollingInterval).Should(BeNumerically(">=", 1))
+		}, timeout, pollingInterval).Should(BeNumerically(">", common.UnusualRestartCountThreshold))
+
+		if utils.IsOpenshift(f.K8sClient) {
```

Comment:
Are we sure this is openshift only, we could enable prometheus on kubevirtci and get the same information I think.

Reply:
The reason I opted in for the skipNotOpenShift approach is that we didn't have 1.21 lanes,
plus to spare some hacking by grabbing the route. but it will for sure be possible to have these running on k8s as well

Reply:
I would create a funcion `isMonitoringEnabled` or `isPrometheusAvailable`, and if it is hard to correctly check for monitoring/prometheus now, you can just do `utils.IsOpenshift(f.K8sClient)` inside the implementation and leave your description as a comment.
Then you clearly state the intent, and it is easy to find and upgrade the implementation in the future.

Reply:
Done, thx